CondFormats/SiPixelTransient/src/SiPixelTemplate2D.cc

```diff
@@ -64,6 +64,7 @@ bool SiPixelTemplate2D::pushfile(int filenum, std::vector< SiPixelTemplateStore2
    // Add template stored in external file numbered filenum to theTemplateStore
    
    // Local variables
+   char c;

```

Comment:
You are reverting here a commit done by @pmaksim1 in the previous PR : https://github.com/cms-sw/cmssw/commit/fc3c83973262e0007471d8c5710abf0f9a4acc2a#diff-d810f8d340a82d8988d1d72ddfeef082

Please make sure that you work on top of the code which is in the master!
(And in this particular case please revert)